Dockerfile

```diff
@@ -481,6 +481,8 @@ ENV NF_IMAGE_TAG ${NF_IMAGE_TAG:-latest}
 ARG NF_IMAGE_NAME
 ENV NF_IMAGE_NAME ${NF_IMAGE_NAME:-focal}
 
+# CI signal, we set it at image level so that any Buildbot command will inherit this flag
+ENV NETLIFY=true
```

Comment:
I believe this should work, but we should double-check in the buildbot PR upgrading the `build-image` that the `NETLIFY` environment variable is set to `true` inside the build command.

Reply:
Yep agree 👍 in fact I plan to test out a pre-release here, run this through a buildbot release without actually releasing a new `build-image`. I'll update this PR once I have some results.